sdk/core/azure-core-test/README.md

```diff
@@ -39,7 +39,7 @@ Use [TestBase][TestBase.java] to easily create live and playback test cases. Ext
 // Set the AZURE_TEST_MODE environment variable to either PLAYBACK or RECORD to determine if tests are playback or
 // live. By default, tests are run in playback mode.
 public class SessionTests extends TestBase {
-    @Test
+    // Use your JUnit or TestNG annotation here for your testcase
```

Comment:
To be very clear why this changed. The README.md files are being rendered into html and added to the landing page for a given artifact. Javadocs does not like or recognize @Test notation and this would actually cause the javadoc build to fail. There is no way around this other than turning off doclint which, frankly, is a pretty bad idea. Also, trying to add a custom tag prevents the failure but javadoc stops processing a given document which means, quite literally, it cuts off adding the html to the landing page when it encounters one.